src/pages/event/[id].tsx

```diff
@@ -12,24 +14,22 @@ const EventDetailPage: React.FC = () => {
   const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
-    const fetchEvent = async () => {
-      const response = await fetch(
-        `https://sportujspolu-api.onrender.com/api/v1/events/${id}`
-      );
-      setEvent(await response.json());
-      setIsLoading(false);
+    const loadEvent = async () => {
+      if (id) {
+        const fetchedEvent = await fetchEventDetail(id);
+        setEvent(fetchedEvent);
+        setIsLoading(false);
+      }
     };
```

Comment:
Always prefer early return. 

```suggestion
    const loadEvent = async () => {
      if (!id) {
        return
      }
      
        const fetchedEvent = await fetchEventDetail(id);
        setEvent(fetchedEvent);
        setIsLoading(false);
    };
```

Reply:
resolved